pom.xml

```diff
@@ -1,65 +1,71 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <parent>
-    <groupId>org.jvnet.hudson.plugins</groupId>
-    <artifactId>plugin</artifactId>
-    <version>1.366</version><!-- which version of Hudson is this plugin built against? -->
-    <relativePath>../pom.xml</relativePath>
-  </parent>
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.jenkins-ci.plugins</groupId>
+		<artifactId>plugin</artifactId>
+		<version>4.75</version>
+		<relativePath/>
+	</parent>
+	<properties>
+		<jenkins.version>2.387.3</jenkins.version>
+		<java.level>11</java.level>
+	</properties>
+	<artifactId>copy-data-to-workspace-plugin</artifactId>
+	<version>2.0-SNAPSHOT</version>
+	<name>Copy data to workspace plugin</name>
+	<description>Copies data to workspace directory for each project build</description>
+	<url>https://plugins.jenkins.io/copy-data-to-workspace-plugin/</url>
+	<packaging>hpi</packaging>
 
-  <artifactId>copy-data-to-workspace-plugin</artifactId>
-  <version>1.1-SNAPSHOT</version>
-  <name>Copy data to workspace plugin</name>
-  <description>Copies data to workspace directory for each project build</description>
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/Copy+Data+To+Workspace+Plugin</url>
-  <packaging>hpi</packaging>
-  
-  <distributionManagement>
-    <repository>
-      <id>maven.jenkins-ci.org</id>
-      <url>http://maven.jenkins-ci.org:8081/content/repositories/releases/</url>
-    </repository>
-  </distributionManagement>
-  
-  <developers>
-    <developer>
-      <id>nzhelyakov</id>
-      <name>Nikita Zhelyakov</name>
-      <email>nzhelyakov@gmail.com</email>
-    </developer>
-  </developers>
-  
-  <build>
-    <plugins>
-      <plugin>
-        <artifactId>maven-release-plugin</artifactId>
-        <configuration>
-          <goals>deploy</goals>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
-  
-  <scm>
-    <connection>scm:svn:https://svn.jenkins-ci.org/trunk/hudson/plugins/copy-data-to-workspace-plugin</connection>
-    <developerConnection>scm:svn:https://svn.jenkins-ci.org/trunk/hudson/plugins/copy-data-to-workspace-plugin</developerConnection>
-    <url>https://hudson.dev.java.net/source/browse/hudson/trunk/hudson/plugins/copy-data-to-workspace-plugin</url>
-  </scm>
+	<developers>
+		<developer>
+			<id>nzhelyakov</id>
+			<name>Nikita Zhelyakov</name>
+			<email>nzhelyakov@gmail.com</email>
+		</developer>
+		<developer>
+			<id>jimmyleocn</id>
+			<name>Jimmy Leo</name>
+			<email>taroer@gmail.com</email>
+		</developer>
+	</developers>

```

Comment:
The developers block has been replaced by the entries in the repository permissions updater
```suggestion
```